Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/incomplete-test-creation-dialog.component.ts

```diff
@@ -0,0 +1,31 @@
+﻿import { Component, Inject, OnInit } from '@angular/core';
+import { MdDialog, MdDialogRef, MD_DIALOG_DATA } from '@angular/material';
+import { ActivatedRoute } from '@angular/router';
+import { TestService } from '../tests.service';
+import { Test } from '../tests.model';
+
+@Component({
+    moduleId: module.id,
+    selector: 'incomplete-test-creation',
+    templateUrl: 'incomplete-test-creation-dialog.html'
+})
+
+export class IncompleteTestCreationDialogComponent implements OnInit {
+    isQuestionMissing: boolean;
+    testId: number;
+    test: Test;
```

Comment:
why you declaring the global variable. It's used only in init function. It can be declared inside it.